script/demoScripts/demoMayan.ts

```diff
@@ -0,0 +1,132 @@
+import deployments from '../../deployments/bsc.staging.json'
+import {
+  fetchQuote,
+  getSwapFromEvmTxPayload,
+  Quote,
+} from '@mayanfinance/swap-sdk'
+import { BigNumber, constants } from 'ethers'
+import {
+  MayanBridgeFacet__factory,
+  ILiFi,
+  type MayanBridgeFacet,
+  ERC20__factory,
+  IMayanBridge__factory,
+} from '../../typechain'
+import { ethers, utils } from 'ethers'
+import dotenv from 'dotenv'
+dotenv.config()
+
+const main = async () => {
+  const RPC_URL = process.env.ETH_NODE_URI_BSC
+  const PRIVATE_KEY = process.env.PRIVATE_KEY
+  const LIFI_ADDRESS = deployments.LiFiDiamond
+  const BSC_USDT_ADDRESS = '0x55d398326f99059fF775485246999027B3197955'
+
+  const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
+  const signer = new ethers.Wallet(PRIVATE_KEY as string, provider)
+  const mayan = MayanBridgeFacet__factory.connect(LIFI_ADDRESS, provider)
+  const deadline = Math.floor(Date.now() / 1000) + 60 * 10 // 10 minutes from the current Unix time
+
+  const address = await signer.getAddress()
+
+  let tx
+
+  const quote: Quote = await fetchQuote({
+    amount: 10,
+    fromToken: BSC_USDT_ADDRESS,
+    toToken: '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
+    fromChain: 'bsc',
+    toChain: 'polygon',
+    slippage: 3,
+  })
+
+  const swapFee = getAmountOfFractionalAmount(
+    quote.swapRelayerFee,
+    Math.min(8, quote.fromToken.decimals)
+  )
```

Comment:
Why do we need to do that parsing are the parameters not present in the result of `getSwapFromEvmTxPayload`?